apps/web/app/api/escrow/approve/route.ts

```
@@ -0,0 +1,93 @@
+import { NextRequest, NextResponse } from 'next/server'
+import { createClient } from '@supabase/supabase-js'
+import type { KindFiDB } from '@services/supabase'
+
+async function validateSignatures(
+  signatures: any[],
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)